src/StoreKeeper/WooCommerce/B2C/Commands/CommandRunner.php

```diff
@@ -133,31 +135,39 @@ protected function logVmStats(array $context = [])
     }
 
     /**
-     * @param $name
-     *
-     * @throws SubProcessException
+     * @throws BaseException
      */
     public function executeAsSubProcess(
         string $name,
         array $arguments = [],
-        array $assoc_arguments = [],
-        int $timeout = 0
+        array $assoc_arguments = []
     ): int {
-        $php_script = STOREKEEPER_WOOCOMMERCE_B2C_ABSPATH.'/scripts/run-command-from-input.php';
         $input = self::getSubProcessInputString($name, $arguments, $assoc_arguments);
-        $process = $this->executePhp(
-            [
-                $php_script,
-            ],
-            $input,
-            $timeout
-        );
 
-        return $process->getExitCode();
+        return static::runCommandWithInput($input);
```

Comment:
it does not do any subprocess anymore but the function is called `executeAsSubProcess`, i think the subprocess was still usefull for the wp cli commands, becasue many wordpress plugins have memory leaks. How do they work now?

Reply:
It now spawns sub processes if command is executed through wp-cli and has the `--allow-spawn` argument